tests/kernel/run_kselftests.pm

```diff
@@ -30,13 +30,15 @@ sub prepare_kselftests_from_git
     my $git_tree = get_var('KERNEL_GIT_TREE', 'https://github.com/torvalds/linux.git');
     my $git_tag = get_var('KERNEL_GIT_TAG', '');
     zypper_call('in bc git-core ncurses-devel gcc flex bison libelf-devel libopenssl-devel');
```

Comment:
Perhaps add `kernel-devel` here? So that `/lib/modules/$(uname -r)/build`, which some selftests might need, are always there in the SUT.